src/app/pages/SpotTradingPage/components/TradeDialog/index.tsx

```diff
@@ -0,0 +1,280 @@
+                value={
+                  <>
+                    {weiToNumberFormat(minReturn, 6)}{' '}
+                    <AssetRenderer asset={targetToken} />
+                  </>
+                }
+              />
+            )}
+
+            {orderType === OrderType.LIMIT && (
+              <>
+                {limitPrice && (
+                  <LabelValuePair
+                    label={t(
+                      translations.spotTradingPage.tradeDialog.limitPrice,
+                    )}
+                    value={
+                      <>
+                        {stringToFixedPrecision(limitPrice, 6)}{' '}
+                        <AssetRenderer asset={targetToken} />
+                      </>
+                    }
+                  />
+                )}
+                <LabelValuePair
+                  label={t(translations.spotTradingPage.tradeDialog.duration)}
+                  value={
+                    <>
+                      {!duration ? (
+                        <span>&#8734;</span>
+                      ) : duration === 1 ? (
+                        t(
+                          translations.spotTradingPage.limitOrderSetting
+                            .day_one,
+                          { count: duration },
+                        )
+                      ) : (
+                        t(
+                          translations.spotTradingPage.limitOrderSetting
+                            .day_other,
+                          { count: duration },
+                        )
+                      )}
+                    </>
+                  }
+                />
+              </>
+            )}
+          </div>
+
+          <div className="tw-mt-4">
+            {spotLocked && (
+              <ErrorBadge
+                content={
+                  <Trans
+                    i18nKey={translations.maintenance.spotTrades}
+                    components={[
+                      <a
+                        href={discordInvite}
+                        target="_blank"
+                        rel="noreferrer noopener"
+                        className="tw-text-warning tw-text-xs tw-underline hover:tw-no-underline"
+                      >
+                        x
+                      </a>,
+                    ]}
+                  />
+                }
+              />
+            )}
+          </div>
+          <DialogButton
+            confirmLabel={t(translations.common.confirm)}
+            onConfirm={submit}
+            disabled={spotLocked || !connected}
+            cancelLabel={t(translations.common.cancel)}
+            onCancel={onCloseModal}
+            data-action-id="spot-reviewDialog-submit"
+          />
+        </div>
+      </Dialog>
+    </>
+  );
+};
+
+interface IOrderLabelProps {
+  orderType: React.ReactNode;
+  tradeType: React.ReactNode;
+  className?: string;
+}
+
+export const OrderLabel: React.FC<IOrderLabelProps> = ({
```

Comment:
Since this component is used in many places, extract it to its own file.

Reply:
done